Sources/XCRemoteCache/Dependencies/DependenciesReader.swift

```diff
@@ -92,25 +80,99 @@ public class FileDependenciesReader: DependenciesReader {
         return yaml.mapValues { $0.components(separatedBy: .whitespaces) }
     }
 
-    private func readRaw() throws -> [String: String] {
+    func readRaw() throws -> [String: String] {
+        let fileData = try getFileData()
+        let fileString = try getFileStringFromData(fileData: fileData)
+        let yaml = try getYaml(fileString: fileString)
+        return yaml
+    }
+
+    func getFileData() throws -> Data {
         guard let fileData = fileManager.contents(atPath: file.path) else {
             throw DependenciesReaderError.readingError
         }
+        return fileData
+    }
+
+    func getFileStringFromData(fileData: Data) throws -> String {
         guard let fileString = String(data: fileData, encoding: .utf8) else {
             throw DependenciesReaderError.invalidFile
         }
-        // .d matches the .yaml format
```

Comment:
Minor: would be nice to keep that comment. It isn't obvious why parsing .d with yaml